interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,431 @@
+ * @copyright Copyright (c) 2017 MedEx <support@MedExBank.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+
+var labels = [];
+var postcards = [];
+var show_just;
+
+
+$(function () {
+  
+});
+
+
+/*
+ * Function to find a patient in the DB
+ * This pop-up is the standard openEMR file find_patient_popup.php
+ * It returns pid, lname, fname, dob to function "setpatient" below
+ * which then populates the form with the select patient data
+ */
+function recall_name_click(field) {
+    dlgopen('../../main/calendar/find_patient_popup.php?pflag=0', '_blank', 500, 400);
+}
+
+/*
+ * Function to insert patient data into addRecall fields
+ * pid is sent to server for the data to display
+ */
+function setpatient(pid, lname, fname, dob) {
+    var f = document.forms['addRecall'];
+    f.new_recall_name.value = lname + ', ' + fname;//+ '&nbsp; ('+dob+')'+''+pid;
+                                                   //go get the rest of the data
+    top.restoreSession();
+    $.ajax({
+        type: "POST",
+        url: "save.php",
+        data: {
+            'pid': pid,
+            'action': 'new_recall'
+        }
+    }).done(function (result) {
+        obj = JSON.parse(result);
+        if (obj.DOLV > '') {
+            //check to see if this is an already scheduled appt for the future
+            //if so, do you really want a recall? Ask.
+            //Maybe that appt needs to be removed...
+            var now = moment(); //new Date()).format('YYYY-MM-DD'); //todays date
+            var dolv = moment(obj.DOLV); // another date
+            var duration = dolv.diff(now, 'days');
+            if (duration > '0') { //it's a future appt dude!
+                alert(xljs_NOTE + ': ' + xljs_PthsApSched + ' ' + obj.DOLV + '...');
+            }
+        }
+        $(".news").removeClass('nodisplay');
+        $("#new_pid").val(obj.pid);
+        $("#new_phone_home").val(obj.phone_home);
+        $("#new_phone_cell").val(obj.phone_cell);
+        if (obj.hipaa_allowsms === "NO") {
+            $('#new_allowsms_no').prop('checked', true);
+        } else {
+            $('#new_allowsms_yes').prop('checked', true);
+        }
+        if (obj.hipaa_allowemail === 'NO') {
+            $("#new_email_no").prop('checked', true);
+        } else {
+            $("#new_email_yes").prop('checked', true);
+        }
+        if (obj.hipaa_voice === 'NO') {
+            $("#new_voice_no").prop('checked', true);
+        } else {
+            $("#new_voice_yes").prop('checked', true);
+        }
+        $("#new_address").val(obj.street);
+        $("#new_city").val(obj.city);
+        $("#new_state").val(obj.state);
+        $("#new_postal_code").val(obj.postal_code);
+        $("#new_DOB").html(obj.DOB);
+        $("#new_email").val(obj.email);
+        if (obj.DOLV > '') {
+            $("#DOLV").val(obj.DOLV);
+        } else {
+            var today = moment().format('YYYY-MM-DD');
```

Comment:
TODO
Potential issue here.
Also make sure using correct datepicker in this code